src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -127,6 +127,7 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
 
   // Item string changes
   private getItemStringChanges(): ItemStringChanges | undefined {
+    if (this.itemForm.pristine || !this.itemForm.touched) return {};
```

Comment:
why both pristine and not touched?

Reply:
For some reason, on the first loading, the itemForm is marked as dirty. So I had to add the !this.itemForm.touched to handle this case.

Reply:
If marked dirty, why wouldn't the save/cancel display? They are based on dirty. And if the code goes through "markAsPristine" as is not pristine afterwards, something looks suspicious.

Reply:
So in the end, the way the save/cancel display work was conflicting with what I was trying to do. So I've changed the way to check to see if the fields have been changed